tests/Util/FakeLoadBalancer.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Wikibase\TermStore\MediaWiki\Tests\Util;
+
+use InvalidArgumentException;
+use Wikimedia\Rdbms\IDatabase;
+use Wikimedia\Rdbms\ILoadBalancer;
+use Wikimedia\Rdbms\LoadBalancer;
+
+class FakeLoadBalancer extends LoadBalancer {
+
+	/** @var IDatabase */
+	private $dbr;
+
+	/** @var IDatabase */
+	private $dbw;
+
+	/**
+	 * @param array $params should contain 'dbr' and optionally 'dbw' IDatabase instances
+	 */
+	public function __construct( array $params ) {
+		// no parent constructor call, we only use the LoadBalancer class so we don’t have to
+		// override every ILoadBalancer method – they’ll just crash if someone tries to use them
```

Comment:
:thinking: out of scope..  this signals it is about time to apply some ISP on ILoadBalancer